extensions/blocks/contact-info/editor.scss

```diff
@@ -1,5 +1,5 @@
 .jetpack-contact-info-block {
-	padding: 10px 18px;
+	padding: 10px 0;
```

Comment:
Should we drop this padding completely? I don't see columns block having this space for child blocks either.

Additionally setting top/bottom to `0` doesn't affect anything since core doesn't add any padding.

Reply:
It helps a bit with the hovering/selecting of the parent block but frankly that's about it. Maybe you're right and we should just remove it completely.

Reply:
Up to you, I don't feel strongly about it either way. :-) 

It does help with core issue how difficult these inner blocks are to interact with. 😞

Reply:
I'll remove it.

I'd be more beneficial to find a solution for core rather than us adding a hack just for our blocks 😊

Reply:
Done in 8861390

Reply:
Yup! — hooray for consistency. 🎉 🌮